src/Resources/views/base.html.twig

```diff
@@ -127,7 +127,15 @@
                 </div>
 
                 {% block topbar %}
-                    <nav class="whatwedo_crud-topbar whatwedo-utility-topbar">
+                    <nav class="whatwedo_crud-topbar whatwedo-utility-topbar flex flex-row-reverse">
+                        {% block actions %}
+                            {% include '@araiseCrud/includes/topbar/_actions.html.twig' %}
+                        {% endblock %}
+
+                        {% block breadcrumbs %}
+                            {% include '@araiseCrud/includes/topbar/_breadcrumbs.html.twig' %}
+                        {% endblock %}
+
```

Comment:
Don't change the order of the HTML elements for such visible noticable cases. This is confusing to keyboard users and bad for accessibility.

The semantic order of HTML code shouldn't be far from its visible order as the expected behaviour on focus order is what-you-see-is-what-you-get.

Reply:
@renestalder You're right! What the hell was I thinking? 😂